tests/test-conda-env3.yaml

```diff
@@ -7,4 +7,5 @@ dependencies:
 - icu 58.2
 - libgcc-ng=8.2
 - libstdcxx-ng=8.2
+  #- libgfortran-ng=7.3
```

Comment:
Was this not needed?

Reply:
Yes, this is not needed anymore after Bryant's fix.